doc/ddc-path.txt

```diff
@@ -54,6 +54,13 @@ EXAMPLES						*ddc-path-examples*
 ==============================================================================
 PARAMS							*ddc-path-params*
 
+						     *ddc-path-param-absolute*
+absolute	(boolean)
+	If it is true, returns absolute path.
+	Otherwise, returns relative path from the current directory.
```

Comment:
This is the grammartical issue.
1. Add the subject phrase. What returns something?
2. Add the article for each noun phrase, _e.g._, 'the absolute path'

Reply:
Fix it.

Reply:
Thanks!